x/relationships/types/models/Relationship_test.go

```diff
@@ -0,0 +1,86 @@
+package models_test
```

Comment:
I think it's better to rename this file as `relationship_test.go` (all lower case)